src/components/selection-list/selection-list.st.css

```diff
@@ -0,0 +1,52 @@
+@namespace "SelectionList";
+
+:import {
+    -st-from: "../../style/default-theme/variables.st.css";
+    -st-named: borderFocus;
+}
+
+.list {
+    width: 276px;
+    border: value(borderFocus);
+}
+
+.list-item {
+    -st-states: disabled, focused, selected;
+    width: 100%;
+    height: 36px;
+    padding: 6px 24px;
+    font-family: ArialMT;
```

Comment:
I'm reluctant to use the variables from the theme file until we figure out semantics more clearly. A few examples: is it ok to use color `DS2` if it suits my needs, or do I need to create a meaningful alias such as `selectedDisabledHoveredListItemBackgroundColor`? Should I use `color_Active` for text of an active item, or is it reserved for background?

If we start using vars with unclear semantics, it will be harder to fix them in the future. It's much easier to spot and fix a hardcoded `#000` value then to figure out if a particular instance of `colorDefault` variable is used correctly.